app/views/reviews/_review.html.erb

```diff
@@ -1,11 +1,26 @@
-
 <div class="panel">
 
     <div id="info">
       <ul>
-        <ul>Header: <%= review.header  %></ul>
-        <ul>Rating: <%= review.rating  %></ul>
-        <ul>Review: <%= review.body  %></ul>
+        <p>Header: <%= review.header  %></p>
+        <p>Rating: <%= review.rating  %></p>
+        <p>Review: <%= review.body  %></p>
+        <% if current_user %>
+        <div id="<%= review.id %>">
+          <div id="ballotBox">
+            <a id="upvote" class="vote"> <%= image_tag("upvote.png", :class =>"hvr-grow") %> </a>
+            <div id="review_score"> <%= review.score %> </div>
+            <a id="downvote" class="vote"><%= image_tag("downvote.png", :class =>"hvr-grow") %></a>
+          </div>
+        </div>
+        <% else %>
+        <div id="ballotBox">
```

Comment:
Hmm, I'm not sure I'd even want to show upvotes and downvotes to a non-logged-in-user.